flask_debugtoolbar/templates/base.html

```diff
@@ -1,4 +1,4 @@
-<div id="flDebug" style="display:none;">
+<div role="navigation" id="flDebug" style="display:none;">
```

Comment:
Why? It's been years since I actively worked on browser-facing code, so if this is some new HTML thing that I'm now ignorant of, please enlighten me.

Reply:
This can be another separate PR. It's a change to improve the toolbar's accessibility compliance, which is helpful for developers using accessibility tools like a screen reader. HTML5 has native landmark tags (e.g. `<nav>`) to indicate an element's purpose. Using the ARIA `role` attribute is a HTML4 backwards compatible way of doing the same thing.